modules/ting_search/ting_search.module

```diff
@@ -814,6 +837,7 @@ function ting_search_sort_form($form_state) {
  *   Returns an array of sort options.
  */
 function ting_search_sort_options() {
+  // TODO BBS-SAL: Where is the machine-names used?
```

Comment:
Probably in the dropdown in the search interface:

![sog harry potter kobenhavns biblioteker 2017-10-26 11-37-56](https://user-images.githubusercontent.com/73966/32046036-28d67726-ba42-11e7-8886-c9d63e73b379.png)

Reply:
noted!